<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-UNQUKT1" ignored="3" name="Default test(failed)" tests="1" failures="1" timestamp="2022-03-05T16:04:19 IST" time="0.052" errors="0">
  <testcase name="TC06" time="0.009" classname="com.testng.testsuite.SampleB">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at com.testng.testsuite.SampleB.TC06(SampleB.java:14)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- TC06 -->
  <testcase name="TC07" time="0.0" classname="com.testng.testsuite.SampleB">
    <ignored/>
  </testcase> <!-- TC07 -->
  <testcase name="TC05" time="0.0" classname="com.testng.testsuite.SampleB">
    <ignored/>
  </testcase> <!-- TC05 -->
  <testcase name="TC08" time="0.0" classname="com.testng.testsuite.SampleB">
    <ignored/>
  </testcase> <!-- TC08 -->
</testsuite> <!-- Default test(failed) -->
